exporter/signalfxexporter/exporter.go

```diff
@@ -102,6 +103,17 @@ func newSignalFxExporter(
 				// TODO: What other settings of http.Client to expose via config?
 				//  Or what others change from default values?
 				Timeout: config.Timeout,
+				Transport: &http.Transport{
+					Proxy: http.ProxyFromEnvironment,
+					DialContext: (&net.Dialer{
+						Timeout:   5 * time.Second,
```

Comment:
I think default transport's dialer timeout is 30s so it may make more sense to use that and refine as necessary (both usages).

Reply:
I have added suggested changes. Please review the same.